releasetasks/templates/firefox/push_to_releases.yml.tmpl

```diff
@@ -70,7 +70,6 @@
             - index.releases.v1.{{ branch }}.latest.{{ product }}.latest.push_to_cdn
         payload:
             maxRunTime: 7200
-            # TODO - create specific image for this
```

Comment:
I suppose even though we are not replacing this image as it doesn't require antivirus, this comment might still be valid as we probably don't want to use 'kmoir' location forever?